libr/bin/format/coff/coff_specs.h

```diff
@@ -359,6 +396,19 @@ struct xcoff64_scn_hdr {
 	char pad44[4];
 });
 
+#if R_BIN_COFF_BIGOBJ
```

Comment:
I suggest calling the new data structures bigobj_symbol, etc. Having the same struct name have twice different meanings depending on flags is a bit hacky.